ansible/roles/ocp-workload-velero/templates/velero-deployment.yml.j2

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  namespace: velero
+  name: velero
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: velero
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8085"
+        prometheus.io/path: "/metrics"
+    spec:
+      restartPolicy: Always
+      serviceAccountName: velero
+      containers:
+        - name: velero
+          image: quay.io/ocpmigrate/velero:fusor-dev
```

Comment:
I think it'd be worth making this a variable in defaults that can be overridden

Reply:
yeah. that makes sense. Will do that.